README.md

```diff
@@ -83,6 +83,8 @@ If you notice something that you feel is broken or missing in configure feel fre
 |JSON|***[builtin]*** http://github.com/paked/configure |`NewJSON(io.Reader)`| Retrieves values from an `io.Reader` containing JSON |
 |Flag|***[builtin]*** http://github.com/paked/configure |`NewFlag()`| Retrieve flagged values from `os.Args` in a `--x=y` format|
 |HCL|***[builtin]*** http://github.com/paked/configure |`NewHCL(io.Reader)`| Retrieve values from an `io.Reader` containing HCL|
+|MongoDb|***[external]*** https://github.com/LouAdrien/configuremongo |`NewMongo("mongodb://127.0.0.1:27017/test_conf_db","confs")`| Retrieve values from an `io.Reader` containing HCL|
```

Comment:
Just a couple things to fix here:

1. replace `NewMongo("mongodb://127.0.0.1:27017/test_conf_db","confs")` with `NewMongo(dburl, dbname)`
2. add a proper description to replace `Retrieve values from an `io.Reader` containing HCL`. Maybe something like `retrieve values from a MongoDB database`